vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -181,7 +203,7 @@ func (r RefCountsMap) Init(d drivers.VolumeDriver, mountDir string, name string)
 	err = r.discoverAndSync(c, d)
```

Comment:
>Frankly we need two bitmaps - an active one and one to handle collecting refcounts

@govint Yes and I believe @pshahzeb is proposing same;
Active one => Original refCount map
Temporary => reconcile-refmap